app.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+# books = []
+# people = []
+# rentals = []
```

Comment:
- [ Required ] I think it would be nice if you remove any commented pieces of code from your code base, commented code gives your code base an unprofessional look and is also considered a sign of code that is not clean.